server/controllers/ical.coffee

```diff
@@ -27,7 +28,12 @@ createCalendar = (calendarName, callback) ->
         return callback err if err
 
         if events.length > 0
-            calendar.add event.toIcal() for event in events
+            for event in events
+                # first time app opening calendar and new calendar dummy event
+                endDateArray = moment(event.end, 'YYYYMMDD')._a
```

Comment:
You're right about more clarity but the third solution seems to be to much complex and can slow down the performance when exporting a lot of events as there is no function isDummy() in an event object. So I changed to the second one.